src/packages/@ncigdc/modern_components/ConsequencesTable/ConsequencesTable.js

```diff
@@ -130,9 +130,11 @@ export default compose(
       canonicalTranscriptId,
       functionalImpactTranscript,
       theme,
+      context,
     }: TProps = {},
   ) =>
     <LocalPaginationTable
+      entityName={context}
```

Comment:
yeah I thought but that but it looked like maybe something was actually using that default